cibyl/utils/dicts.py

```diff
@@ -20,7 +20,7 @@
 LOG = logging.getLogger(__name__)
 
 
-def subset(dictionary, keys):
+def subset(dictionary, keys: list) -> dict:
```

Comment:
dictionary is missing the annotation, a general `dict` one should be fine for this function, I believe

Reply:
Changed. Thanks!